flashcards.rb

```diff
@@ -0,0 +1,86 @@
+
+class Menu
+  @cards_in_play = []
```

Comment:
I would instead use a class variable for Flashcards that starts out as an empty array, and every time initialize runs in `Flashcard`, push to that array class variable.